core/src/main/resources/hudson/Messages_fr.properties

```
@@ -49,16 +47,82 @@ Util.day   ={0} j
 Util.month ={0} mo.
 Util.year  ={0} an.
 
-UpdateCenter.DownloadButNotActivated=La mise à jour a été téléchargée avec succès. Elle sera activée lors du prochain démarrage
-
-FilePath.TildaDoesntWork=''~'' n''est supporté que sur les shells Unix.
+FilePath.TildaDoesntWork=’~’ n''est supporté que sur les shells Unix.
 
+PluginManager.DisplayName=Plugins
 PluginManager.PortNotANumber=Le port n''est pas un nombre
 PluginManager.PortNotInRange=Le port n''est pas dans l''intervalle de {0} à {1}
-AboutJenkins.DisplayName=A propos de Jenkins
+PluginManager.UploadPluginsPermission.Description=\
+  Obsolète - Veuillez utiliser le droit Overall/Administer à la place
```

```suggestion
  Obsolète - Veuillez utiliser le droit "Overall/Administer" à la place
```
For consistency with other translations including english permission terms.